webpack.config.js

```diff
@@ -124,9 +121,19 @@ module.exports = {
     ],
   },
   plugins: [
-    new MiniCssExtractPlugin({
-      filename: 'css/[name].css',
```

Comment:
define CSS output file in the `css.filename` plugin option to keep all relevant options in one place